src/main/java/use_case/assisted_mode/hit/probability/ProbabilityHitCalculator.java

```diff
@@ -1,25 +1,22 @@
-package use_case.probability.hit;
+package use_case.assisted_mode.hit.probability;
 
 import data_access.GameDataAccessObject;
 import entities.Card;
-import entities.Dealer;
 import entities.Player;
-import entities.UserPlayer;
-import use_case.probability.ProbabilityInteractorInterface;
 
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
 
-import static use_case.probability.ProbabilityConstants.BLACKJACK;
-import static use_case.probability.ProbabilityConstants.BUST;
-import static use_case.probability.ProbabilityConstants.fullDeck;
-import static use_case.probability.ProbabilityConstants.sampleDeck;
-import static use_case.probability.ProbabilityConstants.SCENARIO;
-import static use_case.probability.ProbabilityConstants.WINS;
-import static use_case.probability.ProbabilityConstants.values;
-import static use_case.probability.ProbabilityConstants.INITIALSCORE;
-import static use_case.probability.ProbabilityConstants.ACETOONE;
+import static use_case.assisted_mode.hit.probability.ProbabilityConstants.BLACKJACK;
```

Comment:
IIRC you can replace all these imports with "import static use_case.assisted_mode.hit.probability.ProbabilityConstants.*"

Reply:
At the end checkstyle will flag for star imports.